automation/WinFormsAutomationApp/AuthenticationHelper.cs

```diff
@@ -230,5 +246,61 @@ private static IPlatformParameters GetPlatformParametersInstance(string promptBe
 #endif
             return platformParameters;
         }
+
+        public static string FromDictionaryToJson(this Dictionary<string, string> dictionary)
+        {
+            var kvs = dictionary.Select(kvp => string.Format("\"{0}\":\"{1}\"", kvp.Key, string.Concat(kvp.Value)));
```

Comment:
Use string interpolation to make this string easier to read: 

var kvs = dictionary.Select(kvp => $"\"{kvp.Key}\":\"{kvp.Value}\"");